packaging/docker/Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM centos:7
+LABEL maintainer="tdockendorf@osc.edu; johrstrom@osc.edu"
+
+# setup the ondemand repositories
+COPY files/ondemand-web.repo /etc/yum.repos.d/ondemand-web.repo
+COPY files/ondemand-ci-web.repo /etc/yum.repos.d/ondemand-ci-web.repo
+RUN curl https://yum.osc.edu/ondemand/RPM-GPG-KEY-ondemand -o /etc/pki/rpm-gpg/RPM-GPG-KEY-ondemand
+RUN rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-ondemand
```

Comment:
All unnecessary if you use this: https://yum.osc.edu/ondemand/latest/ondemand-release-web-latest-1-6.noarch.rpm

Reply:
The ci web one doesn't have RPM but others are. also `rpm --import` isn't necessary if you reference the path to GPG key in repo file then do `yum` commands with `-y`.